packages/components/slider/src/slider.tsx

```diff
@@ -488,6 +493,7 @@ export const useSlider = ({
         "aria-valuenow": value,
         "data-active": dataAttr(isDragging && focusThumbOnChange),
         "aria-orientation": orientation,
+        "aria-valuetext": ariaValueText,
```

Comment:
Change `"aria-valuetext": ariaValueText` to `"aria-valuetext": ariaValueText ?? value`